src/test/scala/org/constellation/cluster/E2ETest.scala

```diff
@@ -134,11 +96,11 @@ class E2ETest
 
     val storedSnapshots = allAPIs.map { _.simpleDownload() }
 
-    // messageSim.dumpJson(storedSnapshots)
+    constellationAppSim.dumpJson(storedSnapshots)
 
     // TODO: Move to separate test
 
-    // TODO: This is flaky and fails randomly sometimes
+    // TODO: This is flaky and fails randomly sometimhttps://twitter.com/MikeCamel/status/1099018153563058176es
```

Comment:
lol

Reply:
sry, random crap ended up in some comments, will remove this and below